tests/basic/storage_pool_test.go

```diff
@@ -3879,25 +3912,28 @@ var _ = Describe("{ResizeNodeMaintenanceMode}", func() {
 			log.FailOnError(err, fmt.Sprintf("fail to enter node %s in maintenence mode", stNode.Name))
 			status, err := Inst().V.GetNodeStatus(*stNode)
 			log.InfoD(fmt.Sprintf("Node %s status %s", stNode.Name, status.String()))
+			//wait for 5 minutes before existing maintenance
+			time.Sleep(5 * time.Minute)
+			log.InfoD(fmt.Sprintf("Exiting maintenence mode on node %s", stNode.Name))
+			t := func() (interface{}, bool, error) {
+				if err := Inst().V.ExitMaintenance(*stNode); err != nil {
+					return nil, true, err
+				}
+				return nil, false, nil
+			}
+			_, err = task.DoRetryWithTimeout(t, 15*time.Minute, 2*time.Minute)
+			log.FailOnError(err, fmt.Sprintf("fail to exit maintenence mode in node %s", stNode.Name))
+			err = Inst().V.WaitDriverUpOnNode(*stNode, 2*time.Minute)
+			log.FailOnError(err, fmt.Sprintf("Driver is down on node %s", stNode.Name))
+			dash.VerifyFatal(err == nil, true, fmt.Sprintf("PX is up after exiting maintenance on node %s", stNode.Name))
```

Comment:
first condition checks for error in line 3927.
and 2nd condition checks for error == nil ,

do we need both the conditions here ?

Reply:
fixed